website/cue/reference/components/sinks/base/prometheus_remote_write.cue

```diff
@@ -234,14 +239,31 @@ base: components: sinks: prometheus_remote_write: configuration: {
 		}
 	}
 	compression: {
-		description: "Supported compression types for Prometheus Remote Write."
-		required:    false
+		description: """
+			Compression configuration.
+
+			All compression algorithms use the default compression level unless otherwise specified.
+			"""
+		required: false
 		type: string: {
 			default: "snappy"
 			enum: {
-				gzip:   "Gzip."
-				snappy: "Snappy."
-				zstd:   "Zstandard."
+				gzip: """
+					[Gzip][gzip] compression.
+
+					[gzip]: https://www.gzip.org/
+					"""
+				none: "No compression."
+				zlib: """
+					[Zlib][zlib] compression.
+
+					[zlib]: https://zlib.net/
+					"""
+				zstd: """
+					[Zstandard][zstd] compression.
+
+					[zstd]: https://facebook.github.io/zstd/
+					"""
```

Comment:
It looks like the docs are missing the `snappy` option. It should be automatically added to all components using the `Compression` enum, no?